packages/sync-repo-settings/src/required-checks.json

```diff
@@ -149,6 +135,32 @@
           "Kokoro - Test: Linkage Monitor",
           "cla/google"
         ]
+      },
+      {
+        "repo": "googleapis/java-spanner",
+        "requiredStatusChecks": [
+          "Kokoro - Test: Binary Compatibility",
+          "Kokoro - Test: Code Format",
+          "Kokoro - Test: Dependencies",
+          "Kokoro - Test: Java 11",
```

Comment:
Spanner and Firestore should both require integration tests

Reply:
done